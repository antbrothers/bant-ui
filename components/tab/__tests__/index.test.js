/*
 * @Author: linjianxi
 * @Date: 2020-01-13 18:04:08
 * @LastEditTime : 2020-01-13 18:06:18
 * @Description: file content
 */
import React from "react";
import assert from "power-assert";
import { render, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Tab from "../index";
import { action } from "@storybook/addon-actions";

// import { LoadingIcon } from "../../icon";

describe("<Button/>", () => {
  it("should render a <Button/> components", () => {
    const wrapper = render(
      <div>
        <Tab>默认</Tab>        
      </div>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})