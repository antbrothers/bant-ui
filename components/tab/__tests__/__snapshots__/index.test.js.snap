// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> should render a <Button/> components 1`] = `
<div>
  <div>
    <span>
      默认
    </span>
  </div>
</div>
`;
